README.md

```diff
@@ -46,6 +46,28 @@ The `-p` option can be used to skip loading any plugin modules.
 The `-s` option can be used to only parse PELs if they are for serviceable
 errors, while the `-n` option will only parse PELs for non-serviceable ones.
 
+### Installation
```

Comment:
If there any pre-req modules for installation please add it.

Reply:
`setuptools` and `pip3`, have added instruction on how to install them.